tests/bootstrap.php

```
@@ -0,0 +1,914 @@
+        return [];
+    }
+}
+
+/**
+ * Test-compatible xoops_getHandler() — returns stub handlers without real DB.
+ *
+ * @param string $name
+ * @param bool   $optional
+ * @return XoopsObjectHandler|object|false
+ */
+function xoops_getHandler($name, $optional = false)
```

PHPMD (2.15.0)

[error] The method xoops_getHandler has a boolean flag argument $optional, which is a certain sign of a Single Responsibility Principle violation. (undefined)

(BooleanArgumentFlag)